README.md

```diff
@@ -133,3 +133,92 @@ scripts/update_torch_wheels.sh
 ```
 
 The same script can be run again when you want to update the _PyTorch/TPU_ wheels.
+
+
+
+# Debugging
+
+Sometimes bad things happen and a deeper look into the _PyTorch/TPU_ stack is necessary.
+In order to do that, _PyTorch/TPU_ has a serie of environment variables and function calls
```

Comment:
"serie" -> "series", "serie" is not a dictionary word. Also the other occurrence a few lines below.

Reply:
Done.